app_dart/lib/src/request_handlers/reset_prod_task.dart

```diff
@@ -53,6 +54,9 @@ class ResetProdTask extends ApiRequestHandler<Body> {
     final String owner = requestData[ownerParam] as String ?? 'flutter';
     final String repo = requestData[repoParam] as String ?? 'flutter';
     String commitSha = requestData[commitShaParam] as String ?? '';
+    final Map<String, dynamic> properties =
```

Comment:
What is the use case for configuring this? Shouldn't we be reusing the properties from the previously scheduled build?

Reply:
The engine post-submit tasks sometimes need to override the generated artifacts on retries. Rather than always overriding the artifacts by default the builders raise an exception if the artifacts and we are re-running the build and only succeed if we we manually pass a property telling the builder to override the artifacts.

A second use case is to run builds deleting the caches first. This is useful for the cases when a gclient sync pollutes the source code cache and we need to force a build with a fresh source code checkout.